opm/common/OpmLog/LogBackend.hpp

```diff
@@ -43,9 +43,14 @@ namespace Opm
         /// Configure how formatMessage() will modify message strings.
         void setMessageFormatter(std::shared_ptr<MessageFormatterInterface> formatter);
 
+        /// Get MessageFormatter.
+        std::shared_ptr<MessageFormatterInterface> getMessageFormatter() const;
+
         /// Configure how message tags will be used to limit messages.
         void setMessageLimiter(std::shared_ptr<MessageLimiter> limiter);
 
+        /// Get MessageLimiter.
+        std::shared_ptr<MessageLimiter> getMessageLimiter() const;
```

Comment:
Note that I have requested a major redesign of this, so it may not be worthwhile to spend much review effort at this point.